### PR TITLE
Installation instructions enhancements

### PR DESCRIPTION
**Pull request by SYNchroACK**
1. currently there is 3 installation methods: native packages, pypi and from repository
2. I removed the from repository from INSTALL.md because IMHO it only makes sense to use it for development purposes
3. therefore, development environment should not be documented on INSTALL.md, I just wrote a note on INSTALL.md to link the developers to DEVELOPERS-GUIDE.md
4. Developers-Guide has not the same installation procedure.... and it also assumes that the developer is not being crazy about security/permissions etc on his contained dev environment... so...

### CODE REVIEW

**Comment by SYNchroACK** [comment]
waiting your feedback @wagner-certat

File: docs/Developers-Guide.md [hunk 1]
```diff
@@ -77,21 +78,83 @@ Similarly, if code does not get accepted upstream by the main developers, it is
...
-If you do any changes on setup.py, data files (e.g. example configurations) or add new bots, you need to rerun the installation routine.
+In case you developed a new bot, you need to update your current development installation. In order to do that, please follow this procedure:
+
+
+1. Add the new bot information to `/opt/dev_intelmq/intelmq/bots/BOTS`, not `/opt/intelmq/etc/BOTS`.
+2. Make sure that you have your new bot in the right place and the information on BOTS file is correct.
+3. Execute the following commands:
+
+```bash
+sudo -#!/bin/sh
```
Comment (wagner-certat) [inline docs/Developers-Guide.md]: does not make sense

File: docs/Developers-Guide.md [hunk 2]
```diff
@@ -77,21 +78,83 @@ Similarly, if code does not get accepted upstream by the main developers, it is
...
+
+cd /opt/dev_intelmq
+pip3 install --upgrade -e .
```
Comment (wagner-certat) [inline docs/Developers-Guide.md]: with `--upgrade` that upgrades all the dependencies too. Without it, only intelmq will be reinstalled/upgraded

File: docs/Developers-Guide.md [hunk 3]
```diff
@@ -116,11 +179,9 @@ For example, to run all tests you can use:
 INTELMQ_TEST_DATABASES=1 INTELMQ_TEST_LOCAL_WEB=1 INTELMQ_TEST_EXOTIC=1 nosetests
 ```
 
-### Configuration files
+### Configuration Test files
```
Comment (wagner-certat) [inline docs/Developers-Guide.md]: Please also change it in the TOC

File: docs/Developers-Guide.md (hunk 1 above)
Comment (SYNchroACK) [reply docs/Developers-Guide.md]: fixed

File: docs/Developers-Guide.md (hunk 2 above)
Comment (SYNchroACK) [reply docs/Developers-Guide.md]: fixed

File: docs/Developers-Guide.md (hunk 3 above)
Comment (SYNchroACK) [reply docs/Developers-Guide.md]: fixed

**Comment by wagner-certat** [comment]
Moving this is milestone 1.1 as the base branch is develop, not maintenance.

**Pull request merged**